django/librarian/views.py

```diff
@@ -299,7 +299,7 @@ def poll_status(request, data_source_id, document_id=None):
     documents = Document.objects.filter(data_source_id=data_source_id)
     poll = False
     try:
-        poll = documents.filter(status__in=[IN_PROGRESS_STATUSES]).exists()
+        poll = documents.filter(status__in=IN_PROGRESS_STATUSES).exists()
```

Comment:
This was the actual issue.

Reply:
this works great! I don't have to refresh anymore to see the updated statuses :)